src/app/settings/admin/opa/component.ts

```diff
@@ -11,11 +11,16 @@
 
 import {Component} from '@angular/core';
 import {Context} from '@shared/components/tab-card/component';
+import {getEditionDirName} from '@app/dynamic/common';
 
 @Component({
   selector: 'km-admin-settings-opa',
   templateUrl: './template.html',
 })
 export class AdminSettingsOPAComponent {
   readonly Context = Context;
+
+  isEE(): boolean {
```

Comment:
Can we make this method part of the `@app/dynamic/common` and use it in all other places?

Reply:
Also please change to `isEnterpriseEdition()`.

Reply:
done :)